pkg/admission/controller/controller.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package admission
+package controller
```

Comment:
hmm..Looks like we can move everything under `admission/webhook` pkg ? lets have a chat about it.